boutdata/data.py

```diff
@@ -1172,9 +1304,8 @@ def redistribute(self, npes, nxpe=None, mxg=2, myg=2, include_restarts=True):
                         # FieldPerp?
                         # check is not perfect, fails if ny=nz
                         raise ValueError(
-                            "Error: Found FieldPerp '"
-                            + v
-                            + "'. This case is not currently handled by BoutOutputs.redistribute()."
+                            "Error: Found FieldPerp '{}'. This case is not currently "
```

Comment:
Probably not something for this PR, but `restart.redistribute` can handle `FieldPerp`s -- much of the logic is repeated between `BoutOutputs.redistribute` and the version in `restart`, could we merge them somehow?

Reply:
Those probably would be good targets for refactoring, although I'm not sure how useful `BoutOutputs.redistribute` really is - redistributing `BOUT.dmp.*.nc` files is probably only useful if you want to append to them, but it's probably better to just squash the old dump files and new dump files separately now that `squashoutput` exists. It'd be more useful for `BoutOutputs` to be able to create redistributed restart files from the data in dump files, but `xBOUT` can already do that. Agree it's a thing for a separate PR though!